txscript/standard.go

```diff
@@ -521,8 +521,7 @@ func isNullDataScript(scriptVersion uint16, script []byte) bool {
 	// OP_RETURN followed by data push up to MaxDataCarrierSize bytes.
 	tokenizer := MakeScriptTokenizer(scriptVersion, script[1:])
 	return tokenizer.Next() && tokenizer.Done() &&
-		(IsSmallInt(tokenizer.Opcode()) || tokenizer.Opcode() <= OP_PUSHDATA4) &&
```

Comment:
So this has some implications w/ the rbf coop close spec in LN, and the spec uses a strict interpretation that it's exactly no more than 80 bytes (not even multiple pushes are allowed as bitcoind does today). 

We already have a flag that removes all the runtime policy checks, so perhaps that's sufficient for your use case? 

https://github.com/btcsuite/btcd/blob/1eb974aab6ef11097571f9517b4b6b0c639ab63b/config.go#L157

Reply:
My main motivation was a way around https://github.com/btcsuite/btcwallet/pull/907 because in btcwallet it considers op_returns > 80 bytes to be dust rather than op_returns